packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -434,10 +434,10 @@ export class StripeHandler {
     }
 
     const country = request.app.geo.location?.country || 'US';
-    const ipAddress = request.auth.credentials
-      ? customer?.tax?.ip_address || ''
-      : request.app.clientAddress;
-    const automaticTax = this.automaticTax;
+    const ipAddress = customer?.tax?.ip_address || request.app.clientAddress;
+    const isCustomerTaxed =
+      !customer || customer.tax?.automatic_tax === 'supported';
+    const automaticTax = this.automaticTax && isCustomerTaxed;
```

Comment:
Customer does not always exist when a user is logged in. For a new user, they may be logged in but will not yet have a customer. In this situation the previous code did not work as expected.

Reply:
To add to this - the code around `const automaticTax = this.automaticTax` was also an issue, since we _do not_ want to enable automatic tax whenever the feature flag is on. The feature flag simply enables automatic tax for new customers.

Reply:
Okay thanks for clarifying. IIUC, there are four scenarios that a user can be in (when Stripe Tax is enabled):
1. Brand new FxA user/new Stripe customer
2. Existing FxA user/new Stripe customer
3. Existing FxA user/existing Stripe customer that predates Stripe Tax for new customers.
4. Existing FxA user/existing Stripe customer that postdates (possibly not a word) Stripe Tax for new customers.

We should have test cases for each of these permutations for the `previewInvoice` and `createSubscriptionWithPayPal` (or similar) handlers, ideally with a utility that factors out this logic of whether or not to apply Stripe Tax to the customer. I'm not going to block on that for this specific bugfix, but it's probably worth a follow-up ticket. What do you think?